micronaut-maven-plugin/src/main/java/io/micronaut/maven/MojoUtils.java

```diff
@@ -49,4 +57,40 @@ public static String findJavaExecutable(ToolchainManager toolchainManager, Maven
         }
         return executable;
     }
+
+    public static List<String> computeNativeImageArgs(List<String> nativeImageBuildArgs, String baseImageRun, String argsFile) throws IOException {
```

Comment:
suggestion: Maybe move to a `GraalVMNativeUtils` class to make things cleaner, since it's not really an utility for mojos.

Reply:
Yeah I see your point. Although it's actually 2 mojos calling that utility method.

I'll see how the class looks like before the final release, and potentially find a better place and/or name.